project/iterative_sorting.py

```diff
@@ -5,22 +5,26 @@ def selection_sort( arr ):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
-
-        # TO-DO: swap
-
-
+        for j in range(cur_index, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
 
+        # swap
+        temp = arr[smallest_index]
+        arr[smallest_index] = arr[cur_index]
+        arr[cur_index] = temp
```

Comment:
This is good, but it can also be done on one line in Python, like so: `arr[smallest_index], arr[cur_index] = arr[cur_index], arr[smallest_index]`.